rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { mount } from "@vue/test-utils";
+import RundeckInfo from "./RundeckInfo.vue";
+
+jest.mock("../../../rundeckService", () => ({
+  getRundeckContext: jest.fn().mockReturnValue({ rdBase: "mocked-rdBase" }),
+  url: jest.fn().mockReturnValue("http://localhost"),
+}));
+
+const mountRundeckInfo = async (props?: Record<string, any>) => {
+  return mount(RundeckInfo, {
+    props: {
+      appInfo: { title: "Rundeck", logocss: "some-css" },
+      ...props,
+    },
```

Comment:
By my previous comment in this PR, I'd like to move the props here. Ie:

```suggestion
    props: {
      appInfo: { title: "Rundeck", logocss: "some-css" },
      version: { logo: false,
      logocss: "some-css",
      number: "1.0.0",
      tag: "stable",
      title: "Rundeck", },
      /// add the other props here
      ...props,
    },
```

The goal is to keep tests dry, but also mount the components with everything you need for the tests, while having the ability to override something if needed.